subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/TestLifecycleListener.kt

```diff
@@ -14,8 +15,13 @@ interface TestLifecycleListener {
     )
 
     fun finished(
-        artifacts: Result<File>,
+        result: TestResult,
         test: TestCase,
         executionNumber: Int
     )
+
+    sealed class TestResult {
+        class Complete(val artifacts: Result<File>) : TestResult()
+        class InComplete(val infraError: InfrastructureError) : TestResult()
```

Comment:
```suggestion
        class Incomplete(val infraError: InfrastructureError) : TestResult()
```
If you mean "unfinished"

Reply:
https://github.com/avito-tech/avito-android/pull/898